PhysicsTools/PatAlgos/python/tools/jetCollectionTools.py

```diff
@@ -423,27 +429,64 @@ def addRecoJetCollection(self,
       setattr(proc, jercVar, proc.jercVars.clone(srcJet = selJet))
       currentTasks.append(jercVar)
       #
-      # 
+      # JetID Loose
       #
       looseJetId = "looseJetId{}".format(tagName)
       if looseJetId in self.main:
         raise ValueError("Step '%s' already implemented" % looseJetId)
-      setattr(proc, looseJetId, proc.looseJetId.clone(src = selJet))
-      #
-      # 
+      setattr(proc, looseJetId, proc.looseJetId.clone(
+          src = selJet,
+          filterParams=proc.looseJetId.filterParams.clone(
+            version ="WINTER16"
```

Comment:
Just noticing that for loose jetID the same setting ("WINTER16") is used for '16, '17 and '18

Reply:
This is because the Loose working point is only defined for 2016 and not defined for 2017 and 2018 by JetMET group. Its consistent with the main nanoAOD setup in [jets_cff.py](https://github.com/cms-sw/cmssw/blob/CMSSW_10_2_18/PhysicsTools/NanoAOD/python/jets_cff.py#L57).